src/Microsoft.AspNet.Razor/Parser/HtmlMarkupParser.Block.cs

```diff
@@ -471,11 +471,18 @@ private void BeforeAttribute()
                 return;
             }
 
+            // Minimized attribute
             if (!At(HtmlSymbolType.Equals))
             {
-                // Saw a space or newline after the name, so just skip this attribute and continue around the loop
```

Comment:
In particular, remind the reader what has already been accepted i.e. what the first `Output()` will do.  The second `Output()` should be obvious to all.